src/server/handlers/index.ts

```diff
@@ -36,9 +34,7 @@ export default {
   HealthGet: Health.get,
   MetricsGet: Metrics.get,
   PostConsents: PostConsents.post,
-  RevokeConsent: RevokeConsent.post,
   UpdateConsent: UpdateConsent.put,
-  GenerateChallengeRequest: GenerateChallenge.post,
   VerifyThirdPartyAuthorization: Authorizations.post,
   validationFail: OpenapiBackend.validationFail,
   notFound: OpenapiBackend.notFound,
```

Comment:
what about DELETE handler?

Reply:
is it moved to another story?

Reply:
Yes, as it's new endpoint will implement in new story also need functionality details like database updates.